queries/hoon/highlights.scm

```diff
@@ -0,0 +1,32 @@
+(number) @number
+
+(string) @string
+
+[
+  "("
+  ")"
+  "["
+  "]"
+]  @punctuation.bracket
+
+[
+  (coreTerminator)
+  (seriesTerminator)
+] @punctuation.delimiter
+
+
+(rune) @keyword
+
+(term) @constant
+
+(aura) @constant.builtin
+
+(Gap) @comment
+
+(boolean) @constant.builtin
+
+(date) @constant.builtin
+(mold) @constant.builtin
+(specialIndex) @constant.builtin
+(lark) @operator
+(fullContext) @constant.builtin
```

Comment:
```suggestion
(fullContext) @punctuation.delimiter
```

Reply:
This one is a similar to mold, it is a symbol alias. So I think `@symbol` would be suitable.